docs/general/ad-filtering/create-own-filters.md

```diff
@@ -577,6 +584,12 @@ AdGuard will try to close the browser tab with any address that matches a blocki
 
 :::
 
+:::caution Restrictions
+
+In AdGuard Browser Extension MV3, it cannot be converted to an MV3 declarative rule, but can be implemented on the content-script side.
```

Comment:
What? What kind of restriction is this?

@105th what is the restriction? Is it supported or not?

Reply:
@SeverAnna looks like i placed old info to the table about $popup. currently we convert it - https://github.com/AdguardTeam/tsurlfilter/tree/epic/tswebextension/packages/tsurlfilter/src/rules/declarative-converter#popup. But they would not work in MV3, so we will disable converting of them.
We will try to use it only in our tsurlfilter engine and close new tabs programmatically.